lib/rubocop/cop/mixin/array_syntax.rb

```diff
@@ -8,7 +8,7 @@ module ArraySyntax
       private
 
       def bracketed_array_of?(element_type, node)
-        return false unless node.square_brackets? && node.values.size > 1
+        return false unless node.square_brackets? && node.values.size >= 1
```

Comment:
Good call

Reply:
Though at that point maybe it would be best to just do

`return false if node.values.empty? || !node.square_brackets?`

I always find `unless` starts to give me pause when the condition has more than one expression

Reply:
Actually the one you suggested read the easiest of the various variations I tried.